### PR TITLE
Authoring PAM and other template QE review

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/externaldb-build-proc.adoc [hunk 1]
```diff
@@ -11,61 +11,79 @@ Complete the steps in this build procedure to provide drivers for any of the fol
...
+The build procedure creates a custom extension image that extends the existing {KIE_SERVER} image. You must import this custom extension image into your OpenShift environment and then reference it in the `EXTENSION_IMAGE` parameter.
 
 .Prerequisites
-* You are logged in to your project in the OpenShift environment using the `oc` command as a user with the `cluster-admin` role.
-* For IBM DB2, Oracle Database, or Sybase, you downloaded the JDBC driver from the database server vendor.
+* You are logged in to your OpenShift environment using the `oc` command. Your OpenShift user must have the `registry-editor` role.
+* For Oracle Database or Sybase, you downloaded the JDBC driver from the database server vendor.
+* You have installed the following required software:
+** Docker
+** Cekit version 2 or 3
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/externaldb-build-proc.adoc]: only cekit version 3 is supported.

File: doc-content/enterprise-only/openshift/externaldb-build-proc.adoc [hunk 2]
```diff
@@ -11,61 +11,79 @@ Complete the steps in this build procedure to provide drivers for any of the fol
...
-../build.sh --registry=docker-registry.custom-domain:80
-../build.sh --artifact-repo=/home/builder/drivers --registry=docker-registry.custom-domain:80
+docker images
 ----
---
++
+Note the name of the image that was built, for example, `jboss-kie-db2-extension-openshift-image`, and the version tag of the image, for example, `11.1.4.4` (not the `latest` tag).
++
+. Access the registry of your OpenShift environment directly and push the image to the registry. Depending on your user permissions, you can push the image into the `openshift` namespace or into a project namespace. For instructions about accessing the registry and pushing the images, see https://docs.openshift.com/container-platform/3.11/install_config/registry/accessing_registry.html#access[Accessing the Registry Directly].
+. When configuring your {KIE_SERVER} deployment, set the following parameters:
```
Comment (spolti) [inline doc-content/enterprise-only/openshift/externaldb-build-proc.adoc]: do not forget to instruct the user to use the external-db template, which is the only one which is configured to configure the extension image.

Comment (mramendi) [reply doc-content/enterprise-only/openshift/externaldb-build-proc.adoc]: This configuration is copied into other templates in a template modification procedure that we have in the docs. I'll add something to this effect.

File: doc-content/enterprise-only/openshift/externaldb-build-proc.adoc [hunk 3]
```diff
@@ -11,61 +11,79 @@ Complete the steps in this build procedure to provide drivers for any of the fol
...
 [subs="attributes,verbatim,macros"]
 ----
-../build.sh --artifact-repo=<local_path_or_url>
+make build oracle artifact=/tmp/ojdbc7.jar version=7.0
 ----
-
-Replace `<local_path_or_url>` with the path to the local directory or the URL for the HTTP server where the driver is provided. Example:
-
++
+In this command, replace `/tmp/ojdbc7.jar` with the pathname of the downloaded Oracle Database driver and `7.0` with the version of the driver.
```
Comment (sterobin) [inline doc-content/enterprise-only/openshift/externaldb-build-proc.adoc]: "path name" two words, per IBM. Comes up a few times. Search all instances of "pathname" and replace.

File: doc-content/enterprise-only/openshift/externaldb-build-proc.adoc [hunk 4]
```diff
@@ -11,61 +11,79 @@ Complete the steps in this build procedure to provide drivers for any of the fol
...
+* For Oracle Database or Sybase, you downloaded the JDBC driver from the database server vendor.
+* You have installed the following required software:
+** Docker
+** Cekit version 3
+** The following libraries and extensions for Cekit:
+*** `odcs-client`, provided by the `python3-odcs-client` or similar package
+*** `docker`, provided by the `python3-docker` or similar package
+*** `docker-squash`, provided by the `python3-docker-squash` or similar package
+*** `behave`, provided by the `python3-behave` or similar package
+*** `s2i`, provided by the `source-to-image` or similar package
```
Comment (sterobin) [inline doc-content/enterprise-only/openshift/externaldb-build-proc.adoc]: For each of these, I suggest "by the __ package or similar package" for clarity (assuming each are packages). IBM isn't a fan of referring to files, folders, packages, objects, etc. without saying what it is.

**Comment by spolti** [comment]
looks good to me.

**Pull request merged**